documentation/getting_started_guide/scripts/run_local_server.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# Adapted from https://stackoverflow.com/a/21957017.
```

Comment:
This was required because `python -m http.server` does not allow you to configure CORS headings, and the local instance of TFE would not load the data because of the existing header type.